packages/x-components/src/x-modules/tagging/wiring.ts

```diff
@@ -130,24 +130,31 @@ export const setQueryTaggingInfo = moduleDebounce(
  *
  * @public
  */
-export const trackResultClickedWire = createTrackResultWire('click');
+export const trackResultClickedWire = createTrackWire('click');
+
+/**
+ * Tracks the tagging of the banner.
+ *
+ * @public
+ */
+export const trackBannerClickedWire = createTrackWire('click');
 
 /**
  * Performs a track of a result added to the cart.
  *
  * @public
  */
-export const trackAddToCartWire = createTrackResultWire('add2cart');
+export const trackAddToCartWire = createTrackWire('add2cart');
 
 /**
- * Factory helper to create a wire for the track of a result.
+ * Factory helper to create a wire for the track of a taggable object.
  *
  * @param property - Key of the tagging object to track.
  * @returns A new wire for the given property of the result tagging.
```

Comment:
One detail just for consistency
```suggestion
 * @returns A new wire for the given property of the taggable object.
```

btw, in our side we were calling it 'taggable element' instead of 'taggable object', but we are fine with both 👍🏽